src/client/components/hiluk-menu/hiluk-menu.tsx

```diff
@@ -107,18 +108,20 @@ export class HilukMenu extends React.Component<HilukMenuProps, HilukMenuState> {
         onClick={addEssenceToCollection}>{STRINGS.addToCollection}</li>);
     }
 
-    shareOptions.push(<li
+    shareOptions.push(<CopyToClipboard text={url}>
+      <li
       className="copy-url clipboard"
       key="copy-url"
-      data-clipboard-text={url}
-      onClick={onClose}>{STRINGS.copyUrl}</li>);
+      onClick={onClose}>{STRINGS.copyRelativeTimeUrl}</li>
```

Comment:
As we discussed offline, this link is not always for relative time

Reply:
Fixed.